lunatrace/lunatrace-dev/src/cli.ts

```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright 2022 by LunaSec (owned by Refinery Labs, Inc)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+import { exec } from "child_process";
+import { writeFile } from 'fs/promises';
+
+import { dump } from 'js-yaml';
+
+import {bslDir, hasuraDir, kratosDir, tmuxpConfgFile} from "./constants";
+import {dbUrlEnv} from "./env";
+import {backend, dockerCompose, frontend, hasura, manifestEtl, oathkeeper, sbomEtl} from "./services";
+import {tmuxpConfig, tmuxWindow } from "./tmux";
+
+const servicesWindow = tmuxWindow('services', [
+  oathkeeper,
+  backend,
+  hasura,
+  frontend,
+  dockerCompose
+]);
+
+const etlWindow = tmuxWindow('etl', [
+  manifestEtl,
+  sbomEtl
+]);
+
+const config = tmuxpConfig('lunatrace', [
+  servicesWindow,
+  etlWindow
+]);
+
+function hasuraInit() {
```

Comment:
Do these get called from anywhere currently?  These would go great in a post-checkout hook in the repo, so that as soon as you check out someone branch you get synced up. Then again, maybe thats going to break in a lot of cases.